docs/README.md

```diff
@@ -5,12 +5,18 @@
 | [![Linux build status][1]][2]  | [![Windows build status][3]][4] |
 | [![Linux code coverage][5]][6] |                                 |
 
+| Interact with us!              |
```

Comment:
I meant right-align to the length of *your* original "Interact with us!" header, but ok 😉